homeworks/Aleksandr.Turuta_turuta7/homework_3/rekurs.js

```diff
@@ -0,0 +1,21 @@
+function fun(n) {
+    if (n === 1) { return true; }
+    if (n <= 1) { return false; }
```

Comment:
here you check `if (n === 1)` and the string after you check `if (n <= 1)`, if `n` will be equal to `1`, it will match the first check, so more accurate is to check `if (n < 1)`

also, I think it's even better to check `if (n === 2)` so we don't get `1` in the set

but in general all previous mistakes are fixed, good job!